[ElectrophysiologyBrowser] Adding manual TestPlan (#6446)
aces/Loris: modules/electrophysiology_browser/test/TestPlan.md

## Electrophysiology Browser test plan
	
### A. Electrophysiology Browser front page

> 1. User can load Electrophysiology Browser module front page IFF has either permission:

Reviewer: ```suggestion
1. User can load Electrophysiology Browser module front page if and only if user has either permission:
```

Reviewer: I just realized today what IFF meant in a review comment from Dave so it might be best to use long way of saying it.

Reviewer 2: Huh? I didn't make any review comments today explaining the word..

Reviewer: LOL, I thought I saw your name but maybe not. (I saw that in another PR, not this one, or maybe I just dreamt it and had a sudden revelation of what IFF meant :P)

   * `electrophysiology_browser_view_site` : _"View all-sites Electrophysiology Browser pages"_
   * `electrophysiology_browser_view_allsites` : _"View own site Electrophysiology Browser pages"_

> 2. User can see other sites Electrophysiology datasets IFF has permission `electrophysiology_browser_view_allsites`. User can see only own-site datasets IFF has permission `electrophysiology_browser_view_site`. 

Reviewer: ```suggestion
2. User can see other sites Electrophysiology datasets if and only if user has permission `electrophysiology_browser_view_allsites`. User can see only own-site datasets if and only if user has permission `electrophysiology_browser_view_site`. 
```

3. Test that all Filters work.  

> 4. Test Clear Filters button

Reviewer: ```suggestion
4. Test Clear Filters button.
```

> 5. Test column table is sortable by headers

Reviewer: ```suggestion
5. Test column table is sortable by headers.
```

> 6. Test that Links work, to correct dataset (raw/all types)

Reviewer: ```suggestion
6. Test that Links work and point to correct dataset (raw/all types).
```

> ### B. subpage: Sessions 

Reviewer: ```suggestion
### B. Subpage: Sessions 
```

> 7. Sidebar:  Navigation links work. 

Reviewer: ```suggestion
7. Sidebar: Navigation links work. 
```

> 8. Data table display : information displayed looks decently laid out, not garbled 

Reviewer: ```suggestion
8. Data table display: information displayed looks decently laid out, not garbled.
```

> 9. Click each "Download" button (there should be 5). Check: Does the download button work?  Does the file that is downloaded have greater than 0kb size? Is a different file downloaded by each button? 

Reviewer: ```suggestion
9. Click each "Download" button (there should be 5). Check: Does the download button work? Does the file that is downloaded have greater than 0kb size? Is a different file downloaded by each button? 
```

Reviewer: note: the electrode file might not be available for all dataset. If it is not available then the button is greyed out. Might be good mentioning it?

> 10. Test Breadcrumb link back to Electrophysiology Browser

Reviewer: ```suggestion
10. Test Breadcrumb link back to Electrophysiology Browser.
```

_For extra credit: Verify LORIS Menu permissions_ 

> User can view the top-level LORIS Menu _Electrophysiology_ and Menu item : _Electrophysiology Browser_ IFF user has either permission:

Reviewer: ```suggestion
User can view the top-level LORIS Menu _Electrophysiology_ and Menu item : _Electrophysiology Browser_ if and only if user has either permission:
```
